src/components/ExponentN.jsx

```diff
@@ -0,0 +1,16 @@
+const ExponentN = ({ count, exponent }) => (
+	<div className="exponent-counter-container">
+		<p className="exponent-label">
+			{count}
+			<sup style={{ fontSize: "1.1rem", verticalAlign: "top" }}>{exponent}</sup>
+		</p>
+		<p className="exponent-result">
+			{[...Array(exponent)].map((o, index) => {
```

Comment:
interesante approach 🤔
En casos como este donde el parámetro `o` no se utiliza, existe la convención de llamarlo `_`. Así otros devs entenderán que es un parámetro no utilizado.
```jsx
{[...Array(exponent)].map((_, index) => {
```
La palabra `Array` lo que hace es crear un nuevo array. El argumento será el número de elementos que haya dentro. Inicialmente estarán vacíos.
```jsx
const array = Array(3)

console.log(array) // Esto mostrará => [,,]

Reply:
gracias por los comentarios! en cuanto a lo del ie. [...Array(3)] crea un Array de [undefined,undefined,undefined] que hace posible utilizar map con un número de iteraciones que decides con un variable. Hay otra manera de hacerlo sin utilizar map?, estaba probando con un for-loop pero no lo consegui.